x-pack/metricbeat/module/mssql/_meta/Dockerfile

```diff
@@ -6,4 +6,4 @@ ENV SA_PASSWORD='1234_asdf'
 
 # Use the same healthcheck as the Windows version of the image.
 # https://github.com/Microsoft/mssql-docker/blob/a3020afeec9be1eb2d67645ac739438eb8f2c545/windows/mssql-server-windows/dockerfile#L31
-HEALTHCHECK --interval=1s --retries=90 CMD /opt/mssql-tools/bin/sqlcmd -U SA -P ${SA_PASSWORD} -Q "select 1"
+HEALTHCHECK --interval=10s --timeout=3s --start-period=10s --retries=3 CMD /opt/mssql-tools18/bin/sqlcmd -S localhost -U SA -P ${SA_PASSWORD} -No -Q "SELECT 1" || exit 1
```

Comment:
I looked `-No` up, and it seems to suppress the output. Could we instead `"SELECT 1" > /dev/null`? Just a question, nothing that needs to done. Given that it's a small output, I'm fine with either.